### DISCUSSION TITLE
Run Windows Terminal with an Explicit Path for a Profile File

### DISCUSSION THREAD

**Discussion by RoyiAvital** [microsoft/terminal#16802] [Q&A]
Is there a way to run `wt.exe` with an explicit path to a Windows Terminal profile file?

**Comment by lhecker** [comment]
Could you explain what you mean with a "profile file"? We only have a settings file (and fragments which are merged into the settings file). If you meant the settings file, then no. The `-p` or `--profile` flag for `wt.exe` does not support changing the path to your settings file at the moment.

**Reply by RoyiAvital** [reply]
It is indeed what I meant. Being able to define a path to the settings file.  
Or at least being able to completely define a profile by command line.

Specifically, I'm just after setting the fond of the Terminal by the launcher.
